hw9/tests/test_task92.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from hw9.tasks.task92 import supressor_class, supressor_generator
+
+
+def test_valid_error_class():
+    with supressor_class(IndexError):
+        [][2]
+    assert True
+
```

Comment:
'supressor_class' works in the task file, but doesn't work with the import module. I couldn't find the solution to this problem on the Internet.

Reply:
Fixed it!